### ISSUE TITLE
Improvement .bx--requirement add space for the (*) 

### ISSUE THREAD

**Issue by lnesi** [carbon-design-system/carbon#41]
Is it possible to consider to do the following change so the (*) is automatically separated from its content in bx--requirement

## Title line template: [Title]: Brief description
.bx--form-requirement::before {
    content: "*\00a0";
}
to
.bx--form-requirement::before {
    content: "*\00a0";
    display: inline-block;
    color: #bb0a30;
}
-->

## Detailed description

Describe in detail the issue you're having. Is this a feature request (new component, new icon), a bug, or a general issue?

> Is this issue related to a specific component?

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?

> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

## Steps to reproduce the issue

1. Step one
`
<div class="bx--form-requirement">Required</div>
`
Result
 
<img width="91" alt="screen shot 2018-09-13 at 11 41 59" src="https://user-images.githubusercontent.com/1512641/45483878-2b7ee700-b74a-11e8-844a-f1954680e30c.png">

Ideal result

<img width="127" alt="screen shot 2018-09-13 at 11 39 13" src="https://user-images.githubusercontent.com/1512641/45483892-320d5e80-b74a-11e8-81f3-05dab591815c.png">

**Comment by tw15egan** [comment]
Makes sense, going to move this over to the repo that would contain this change
